charts/kyuubi/templates/kyuubi-servicemonitor.yaml

```diff
@@ -28,7 +28,7 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: {{ .Release.Name }}
+      {{- include "kyuubi.selectorLabels" $ | nindent 6 }}
```

Comment:
I saw other places use `{{- include "kyuubi.selectorLabels" . | nindent 6 }}`, a silly question, what's the difference between `.` and `$`

Reply:
yeah no differences in this case I believe

generally `$` is about to pass additional data to the template from global context in case the included template uses it, other wise `.` only passes local context defined within that yaml file

Reply:
sounds like either is fine, but let's keep things unified

Reply:
Thanks, updated